src/System.Runtime.Loader/tests/RefEmitLoadContext/RefEmitLoadContextTest.cs

```diff
@@ -123,6 +123,20 @@ public static void LoadRefEmitAssemblyInLoadContext(AssemblyLoadContext loadCont
 
             // Load context of the statically loaded assembly is the custom load context in which dynamic assembly was created
             Assert.Equal(loadContextRefEmitAssembly, AssemblyLoadContext.GetLoadContext((Assembly)asmRefEmitLoadedStatic));
+
+            // Enumerate the assemblies in the AppDomain and confirm that the Dynamically generated assembly is present.
+            var allAssemblies = AppDomain.CurrentDomain.GetAssemblies();
+            bool fDynamicAssemblyFound = false;
+            foreach (Assembly asm in allAssemblies)
+            {
+                if (asm.FullName == asmRefEmitLoaded.FullName)
```

Comment:
Do we actually want to verify that it is exactly same assembly, not just that the full name matches?